EssentialFeed/EssentialFeediOS/Feed UI/Controllers/FeedImageCellController.swift

```diff
@@ -11,26 +11,32 @@ public protocol FeedImageCellControllerDelegate {
 }
 
 public final class FeedImageCellController: FeedImageView {
-	private let delegate: FeedImageCellControllerDelegate
+	private let cellControllerDelegate: FeedImageCellControllerDelegate
```

Comment:
The delegate shouldn't have been renamed.

```suggestion
	private let delegate: FeedImageCellControllerDelegate
```